frontend/src/app/help/page.tsx

```diff
@@ -0,0 +1,54 @@
+import { Metadata } from "next";
+import type { GetStaticProps, NextPage } from "next";
+import { PROCESS_CRUMBS } from "src/constants/breadcrumbs";
+
+import { useTranslation } from "next-i18next";
+import { getTranslations, getMessages } from "next-intl/server";
+
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+
+import Breadcrumbs from "src/components/Breadcrumbs";
+import PageSEO from "src/components/PageSEO";
+import BetaAlert from "src/components/AppBetaAlert";
+import ProcessInvolved from "src/pages/content/ProcessInvolved";
+
+interface RouteParams {
+  locale: string;
+}
+
+import {
+  NextIntlClientProvider,
+  useMessages,
+  useTranslations,
+} from "next-intl";
+
+
+
+export async function generateMetadata({ params }: { params: RouteParams }) {
+  const t = await getTranslations({ locale: params.locale });
+  const meta: Metadata = {
+    title: t("Process.page_title"),
+  };
+  return meta;
+}
+
+export default async function Help() {
```

Comment:
You use `useTransations()` and `useMessages()` if it is a sync function.

Reply:
Believe all `use*` hooks should also only be used by client components.